client/src/App.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react';
+import {
+  Route, Routes, useLocation, useNavigate,
+} from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchAllHomes } from './redux/homesSlice';
+import Layout from './components/Layout';
+import Home from './components/Home';
+import Reserve from './components/Reserve';
+import Reservations from './components/Reservations';
+import AddHome from './components/AddHome';
+import RemoveHome from './components/RemoveHome';
+import HomeDetails from './components/HomeDetails';
+import Login from './components/Login';
+import Register from './components/Register';
+import Splash from './components/Splash';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this main `App` component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

- [ ] OPTIONAL: Refactor the rest of the functions (I found 11)

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/peterdtitan/BnB-homes/assets/57421823/7017d141-9104-45a9-97b0-56231218e8e2)


That way, your code will meet the latest standards. 💪